usaspending_api/api_docs/api_documentation/financial_spending/major_object_class.md

```diff
@@ -0,0 +1,66 @@
+
+## Financial Spending Major Object Class
+**Route**: `/api/v2/financial_spending/major_object_class?fiscal_year=[year]&funding_agency_id=[id]`
+
+Retrieve Award Spending Amounts for all Recipients with Respective Top Tier and Sub Tier Agencies
```

Comment:
Description doesn't match endpoint functionality. Should be: `Retrieve award obligation amounts broken down by major object class`

Reply:
nice catch.